sdb/commands/stacks.py

```diff
@@ -52,6 +52,78 @@
 # future iterations.
 #
 class Stacks(sdb.Command):
+    """
+    Print the stack traces for the active tasks / threads
+
+    By default, the command will aggregate similar call stacks
+    printing them in descending order of frequency. The output
+    includes the `struct task_struct` address, thread state, and
+    aggregation count.
+
+    Optionally, the command can filter stacks, displaying only
+    those that match a given thread state, containing a given
+    function, or belonging to a given kernel module.
```

Comment:
We should add another level of indentation here, so it aligns better when printed via the `help` command.

Reply:
Fixed